### PR TITLE
add redirects field in Query in Mediawiki-fluent-api

### PR DESCRIPTION
**Pull request by sqreek**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'java.util.ArrayList'.](https://github.com/Wikia/pandora/blob/38d3d0eff9fcac7533d72927215faeeb620a4387/lib/mediawiki-fluent-api/src/main/java/com/wikia/mwapi/domain/Query.java#L5) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)
* MAJOR [Use isEmpty() to check whether the collection is empty or not.](https://github.com/Wikia/pandora/blob/38d3d0eff9fcac7533d72927215faeeb620a4387/lib/mediawiki-fluent-api/src/main/java/com/wikia/mwapi/domain/Query.java#L54) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)

File: lib/facade-gateway/src/test/java/com/wikia/pandora/gateway/mediawiki/MediawikiGatewayIntegrationTest.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
   public static final int LIMIT_CATEGORIES_FROM_WIKIA = 5;
   public static final String EMPTY_OFFSET = "";
   public static final long REVISION_ID = 1945L;
-  public static final String AUTHOR_SQREEK = "Sqreek";
+  public static final String AUTHOR_SQREEK = "Sqreek_";
   public static final String IMAGE_TEST_ARTIClE = "Image test";
```
Comment (WikiaTech) [inline lib/facade-gateway/src/test/java/com/wikia/pandora/gateway/mediawiki/MediawikiGatewayIntegrationTest.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by pchojnacki** [comment]
LGTM

**Pull request merged**